### PR TITLE
fix: fix resource leak

### PR DESCRIPTION
**Pull request by will-2012**
### Description

Fix resource leak.

### Rationale

1.Resource manager deadlock can lead to resource leaks and requests getting stuck.
2.Authenticator client connection leak.

### Example

N/A.

### Changes

Notable changes: 
* Manager: remove redundant db load function.
* Resource manager: fix deadlock.
* Authenticator: fix connection leak.
* Others: polish some log.

### CODE REVIEW

File: base/gfspvgmgr/virtual_group_manager.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ const (
 
 var (
 	ErrFailedPickVGF = gfsperrors.Register(VirtualGroupManagerSpace, http.StatusInternalServerError, 540001,
-		"failed to pick virtual group family, need creating global virtual group")
+		"failed to pick virtual group family, need create global virtual group")
```
Comment (sysvm) [inline base/gfspvgmgr/virtual_group_manager.go:32]: fix `need create` as `fix need to create`

File: base/gfspvgmgr/virtual_group_manager.go [hunk 2]
```diff
@@ -29,11 +29,11 @@ const (
...
 	ErrFailedPickGVG = gfsperrors.Register(VirtualGroupManagerSpace, http.StatusInternalServerError, 540002,
-		"failed to pick global virtual group, need staking more storage size")
+		"failed to pick global virtual group, need stake more storage size")
```
Comment (sysvm) [inline base/gfspvgmgr/virtual_group_manager.go:34]: ditto

File: base/gfspvgmgr/virtual_group_manager.go [hunk 3]
```diff
@@ -29,11 +29,11 @@ const (
...
 	ErrStaledMetadata = gfsperrors.Register(VirtualGroupManagerSpace, http.StatusInternalServerError, 540003,
-		"metadata is staled, need forcing refresh metadata")
+		"metadata is staled, need force refresh metadata")
```
Comment (sysvm) [inline base/gfspvgmgr/virtual_group_manager.go:36]: ditto

**Review by sysvm — APPROVED** [review]
LGTM

**Review by ruojunm — APPROVED** [review]

File: base/gfspvgmgr/virtual_group_manager.go (hunk 1 above)
Comment (will-2012) [reply base/gfspvgmgr/virtual_group_manager.go]: Fixed.

File: base/gfspvgmgr/virtual_group_manager.go (hunk 2 above)
Comment (will-2012) [reply base/gfspvgmgr/virtual_group_manager.go]: Fixed.

File: base/gfspvgmgr/virtual_group_manager.go (hunk 3 above)
Comment (will-2012) [reply base/gfspvgmgr/virtual_group_manager.go]: Fixed.

**Pull request merged**